src/main/webapp/app/exercises/modeling/participate/modeling-submission.component.ts

```diff
@@ -549,7 +549,7 @@ export class ModelingSubmissionComponent implements OnInit, OnDestroy, Component
      * The exercise is still active if it's due date hasn't passed yet.
      */
     get isActive(): boolean {
-        return this.modelingExercise && !this.examMode && (!this.modelingExercise.dueDate || moment(this.modelingExercise.dueDate).isSameOrAfter(moment()));
+        return this.modelingExercise && !this.examMode && (!this.modelingExercise.dueDate || dayjs(this.modelingExercise.dueDate).isSameOrAfter(dayjs()));
```

Comment:
```suggestion
        return this.modelingExercise && !this.examMode && (!this.modelingExercise.dueDate || !dayjs(this.modelingExercise.dueDate).isBefore(dayjs()));
```
To stay consistent with the other change

Reply:
I added this discussion point to the [confluence page](https://confluence.ase.in.tum.de/display/ArTEMiS/Code+Review), as I saw that we are using both versions.